python/ray/tune/schedulers/hyperband.py

```diff
@@ -219,7 +223,7 @@ def choose_trial_to_run(self, trial_runner):
         then look into next iteration.
         """
 
-        for hyperband in self._hyperbands:
+        for hyperband in self._get_non_empty_hyperbands():
```

Comment:
nit: to make things a bit more readable, can we do something like:

```python
for hyperband in self._hyperbands:
    # band will have None entries if no resources are to be allocated to that bracket.
    scrubbed = [b for b in hyperband if b is not None]
    for bracket in sorted(scrubbed, key=lambda b...
```

Reply:
If the helper function is not likely to be reused, this can be done in place. In fact, there is already place like `debug_string` could use the function. It depends on whether we want to print a `None` bracket or not. What's your suggestion?